static/src/stylesheets/module/_tls-warning.scss

```diff
@@ -0,0 +1,83 @@
+$tls-warning-rule: darken($garnett-nav-background, 20%);
+
+.tls-warning {
+    background-color: $garnett-nav-background;
+    color: $neutral-1;
+    border-bottom: 1px solid $tls-warning-rule;
+
+    &__browser-links {
```

Comment:
using the `&` makes it very hard to grep for the class in future, we normally warn against this?

Reply:
Good call, trying to be clever!